python/az/aro/azext_aro/_help.py

```diff
@@ -63,6 +63,14 @@
     text: az aro list-credentials --name MyCluster --resource-group MyResourceGroup
 """
 
+helps['aro list-admin-credentials'] = """
```

Comment:
Wondering if we should name it something like `list-admin-kubeconfig` or `get-admin-kubeconfig`. We don't have to follow API naming in CLI.

Reply:
Changed.